### PR TITLE
Add a "Support" link to the Plugin action links

### PR DESCRIPTION
**Pull request by mikeyarce**
Add a new link called "Support" to the /plugins page which will link to the in-dashboard Debugger and allow users to troubleshoot their site and contact Jetpack Support.

### CODE REVIEW

**Comment by georgestephanis** [comment]
I don't think the debugger should be a full menu item.

**Comment by georgestephanis** [comment]
I do wonder, though, if it would be worth directly linking them to jetpack.me/contact-support -- perhaps with blog_id prefilled.

**Comment by chaselivingston** [comment]
We often have to ask the users to go through the process to send us the report from this debugger anyway, so it would be nice to have an easy way for them to generate and send this along with their question first, rather than having to ask for it in a reply.

File: class.jetpack.php [hunk 1]
```diff
@@ -2728,6 +2728,7 @@ function plugin_action_links( $actions ) {
 			return array_merge(
 				$jetpack_home,
 				array( 'settings' => sprintf( '<a href="%s">%s</a>', Jetpack::admin_url( 'page=jetpack_modules' ), __( 'Settings', 'jetpack' ) ) ),
+				array( 'settings' => sprintf( '<a href="%s">%s</a>', Jetpack::admin_url( 'page=jetpack-debugger '), __( 'Support', 'jetpack' ) ) ),
```
Comment (chaselivingston) [inline class.jetpack.php]: Should this be `array( 'support' => sprintf( '<a href="%s">%s</a>', Jetpack::admin_url( 'page=jetpack-debugger '), __( 'Support', 'jetpack' ) ) ),` instead, or am I crazy?

**Comment by mikeyarce** [comment]
@georgestephanis we wouldn't be adding a full menu item, just this link here:
screen shot 2015-05-01 at 10 49 22 am

File: class.jetpack.php (hunk 1 above)
Comment (mikeyarce) [inline class.jetpack.php]: You are probably right, but I'm not sure.

**Comment by georgestephanis** [comment]
I don't think support is the right label for this, if it's just linking to the Debug screen.  Maybe just calling it Debug?  Unsure.

**Comment by dereksmart** [comment]
This makes sense, as there are plenty of opportunities for them to nav to support pages/forms from that page.  Merging

**Pull request merged**